apps/survey/src/views/survey/recall.vue

```diff
@@ -20,10 +21,17 @@ export default defineComponent({
 
     survey.validateState(survey.data, survey.parameters?.session);
 
-    if (survey.recallAllowed)
+    if (survey.recallAllowed) {
```

Comment:
imo this will still fire on every "survey-recall" page visit (which might already be tracked by router-fired events?), so not really a true "recall start", but it's fine if it is intentional